### DISCUSSION TITLE
Some questions with the 1.19 version

### DISCUSSION THREAD

**Discussion by IroPlisk** [MrTJP/ProjectRed#1857] [Q&A] [answered]
So i recently saw the release get pushed, but of the things im going to talk about here relative to the latest beta have not been mentioned in the changelog (i think, unless i missed something), thus i think it is equivalent anyways:

1) Werent there solar panels in this mod? Like i remember some of the first alphas/betas of 1.19 had em, might be wrong since i was jumping between 1.4.7 and 1.19, but is there only one way to generate electric current in this mod (the electrotine gen).

2) I think this point is more of an issue than a Q&A, but the fabrication tables (project table, litography, etc..) have no correct tool. I tried with any type of pickaxe and axe to no avail, evem the electric screwdriver isnt working. Am i missing something?

3) Yet again this is last point is more of an idea or proposal than anything, but what about the Fortran pc? Is there a plan or idea to ever add it back? I guess it's more nostalgia of finding that damned FORTH disc in the mines, but for the little time i have actually used them i had tons of fun.

**Comment by MrTJP** [comment, marked as answer]
1. Solar panels (as well as other blocks from expansion) are not yet re-implemented. Some of these blocks are being re-implemented now.
2. Not sure how this has gone unnoticed for so long, but PR for fix is uploaded. Will push a new beta to Curse tonight (https://github.com/MrTJP/ProjectRed/pull/1858)
3. Would love to but no plans on adding that at the moment. Those computers were *very* complex. They actually emulated a full-blown microprocessor and I believe Eloraam had written major parts of that operating system herself.

**Reply by Tampa** [reply]
Last release for 19.2 has not received the missing blocks yet. Will they be backported to it or should we look for power conversion mod instead?
